dev/PushNotifications/PushNotificationsLongRunningTask/platform.h

```diff
@@ -0,0 +1,35 @@
+﻿#pragma once
+
+#include "../PushNotifications-Constants.h"
+
+#include "PlatformLifetimeTimerManager.h"
+
+struct __declspec(uuid(PUSHNOTIFICATIONS_IMPL_CLSID_STRING)) NotificationsLongRunningPlatformImpl WrlFinal :
+Microsoft::WRL::RuntimeClass<
+    Microsoft::WRL::RuntimeClassFlags<Microsoft::WRL::ClassicCom>,
+    INotificationsLongRunningPlatform,
```

Comment:
You can use `winrt::implements<INotificationsLongRunningPlatform>` and avoid mixing WRL and C++/WinRT.

Reply:
Great feedback! Thank you 😊